source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm

```diff
@@ -0,0 +1,43 @@
+//
+//  CustomInputNumberRenderer
+//  CustomInputNumberRenderer.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "CustomInputNumberRenderer.h"
+#import <AdaptiveCards/ACFramework.h>
+
+@implementation CustomInputNumberRenderer
+
++ (CustomInputNumberRenderer *)getInstance
+{
+    static CustomInputNumberRenderer *singletonInstance = [[self alloc] init];
+    return singletonInstance;
+}
+
+//+ (ACRCardElementType)elemType;
```

Comment:
Remove commented-out code.

Reply:
thanks

---
In reply to: [171734597](https://github.com/Microsoft/AdaptiveCards/pull/1164#discussion_r171734597) [](ancestors = 171734597)